docs/quickstart/android.md

```diff
@@ -167,11 +167,10 @@ Just like we did before, from the `examples` directory:
 
 ## What's next
 
-- Read a full explanation of this example and how gRPC works in our
-  [Overview](http://www.grpc.io/docs/)
+- Read a full explanation of how gRPC works in [What is gRPC?](../guides/) and [gRPC Concepts](../guides/concepts.html)
```

Comment:
could you break the line around the 80th character? It makes it easier to edit in some environments. Same for all the other long lines in this PR.